modules/api/src/main/java/org/eclipse/xpanse/api/exceptions/handler/CommonExceptionHandler.java

```diff
@@ -170,4 +171,16 @@ public Response handleMethodArgumentTypeMismatchException(
         return Response.errorResponse(ResultType.UNPROCESSABLE_ENTITY,
                 Collections.singletonList(ex.getMessage()));
     }
+
+    /**
```

Comment:
please add this to DeploymentExceptionHandler  class. same for the test class also.

Reply:
fixed.